docs/web-sdk.md

```diff
@@ -61,6 +61,29 @@ And then loaded in your JavaScript and bundled as you like:
 import '@exmg/livery';
 ```
 
+#### Using with Next.js
+Next.js makes use of server side rendering, which the Livery Web SDK does not currently support. However, users of Next.js can still use the SDK by making use of Next.js’s support for [dynamic imports](https://nextjs.org/docs/advanced-features/dynamic-import), which ensure the module being imported is evaluated at runtime on the client. The code below illustrates basic usage with Next.js. 
+
```

Comment:
For my taste this example and the text above are both a little lengthy and repetitive with the HTML that's already documented under Basic Usage etc. How about something more succinct like this instead?

> Next.js makes use of server side rendering, which is currently not supported by this SDK. To import our [NPM](#npm) installed SDK module client-side only you can use [useEffect()](..link-to-doc..) (e.g: `useEffect(() => { import('@exmg/livery'); }, []);`) in your component instead of just importing it at the top.

If we do wish to provide more expansive documentation for this then I'd suggest we create a separate "Integrations" section at the end below "Exports" so as to not move the more generally relevant documentation out of view.